lib/server/index.js

```diff
@@ -223,6 +227,11 @@ Server.prototype = {
     var config = this.config;
     var test_page = config.get('test_page');
 
+    // If there is an array of test pages, we cycle through them as the default
+    if (Array.isArray(test_page)) {
+      test_page = test_page[this.testPageCount % test_page.length];
+    }
```

Comment:
@johanneswuerbach not quite sure I understand what you mean. The idea here is that each browser launcher should visit each page once. So the first instance of a browser will visit `test_page[0]`, the second will visit `test_page[1]`, and so forth.

I'm not super happy with this solution, but I am unsure of a better way to tell the server what test page should be loaded for a given browser instance.